src/hubAuth.ts

```diff
@@ -23,6 +23,38 @@ enum AuthStrategy {
 
 const DEFAULT_INSTANCE_URL = 'https://login.salesforce.com';
 
+/**
+ * Function examines the env var TESTKIT_JWT_KEY to determine if it needs to be
+ * reformatted so when saved to a file the RSA key file contents are formatted
+ * properly.
+ *
+ * Throws an error if function is called and the env var is undefined
+ *
+ * returns a string that complies with RSA private key file format
+ */
+const formatJwtKey = (): string => {
+  if (env.getString('TESTKIT_JWT_KEY')) {
+    let keyLines = env.getString('TESTKIT_JWT_KEY', '').split('\n');
+    if (keyLines.length <= 1) {
+      const footer = '-----END RSA PRIVATE KEY-----';
+      const header = '-----BEGIN RSA PRIVATE KEY-----';
+      // strip out header and footer
+      const newKeyContents = env.getString('TESTKIT_JWT_KEY', '').replace(header, '').replace(footer, '');
+      // check to see if newlines were replaced with spaces
+      keyLines = newKeyContents.trim().split(/\s/);
+      if (keyLines.length <= 1) {
+        // still one big string, split into 64 byte chucks
+        keyLines = [header, ...(newKeyContents.match(/.{1,64}/g) as string[]), footer];
+      } else {
+        keyLines = [header, ...keyLines, footer];
+      }
+    }
+    return keyLines.join('\n');
+  } else {
+    throw new Error('env var TESTKIT_JWT_KEY is undefined');
```

Comment:
Throwing for now, since this function should only be called when one has a jwt key env var.